docker/README.md

```diff
@@ -0,0 +1,36 @@
+# Official Docker Images
+
+The latest [official Docker images](https://hub.docker.com/u/roclang) are available on the Docker Hub.
+
+For example, to pull the latest `nightly-ubuntu-2204` image from the Docker Hub, run the following command in a terminal (assuming you have already installed [Docker](https://docs.docker.com/engine/install/)):
+
+```bash
+docker pull roclang/nightly-ubuntu-2204
+```
+
+To start an interactive container based on this image, run:
+
+```bash
+docker run -it --rm roclang/nightly-ubuntu-2204
+```
+
+Note: the `--rm` option will delete the container when you exit. Remove this option is you want to keep the container instead.
```

Comment:
Should we really include `--rm` here? I suspect many people may run the command, play with roc and never read that note. On exit they may lose all their code without knowing it.